### ISSUE TITLE
Email search first for @ in search terms

### ISSUE THREAD

**Issue by azon2111** [TechToThePeople/eu.tttp.qlookup#2]
Thanks for making the module.

I might also suggest a search on .XXX for email first as well as @XXX

**Comment by tttp** [comment]
glad you like it.

what would be the use case for .xxx? find all the .org for instance?

**Comment by azon2111** [comment]
Yeah but not really useful use case in quick search. I would ignore that comment part, lol.
